reactor-core/src/test/java/reactor/core/publisher/FluxAutoConnectTest.java

```diff
@@ -43,42 +43,42 @@ public void constructors() {
 	
 	@Test
 	public void connectImmediately() {
-		FluxProcessor<Integer, Integer> e = Processors.multicast();
+		Sink<Integer> e = Sinks.multicast();
```

Comment:
since the sink isn't really used, can also be replaced with `TestPublisher` which makes it easier to assert subscribers:

```java
		TestPublisher<Integer> testPublisher = TestPublisher.create();

		testPublisher.flux().publish().autoConnect(0, cancel::set);
		testPublisher.assertSubscribers();
		
		cancel.get().dispose();
		testPublisher.assertNoSubscribers();
```